### DISCUSSION TITLE
SunbirdEd Portal - Form Management Tool

### DISCUSSION THREAD

**Discussion by rajeshkumaravel** [Sunbird-Ed/Community#300] [Contribution]
Currently managing (listing all and updating) of forms is being managed by Postman API client.

Portal team has come up with a new tool which will make it easier for us to list, edit, and manage forms within our organisation. We believe this new tool will help streamline our form management process and improve our overall efficiency.

The platform can be accessed with valid domain and bearer token, domain refers to the environment URL. Phase 1 of the tool includes the ability to list forms and edit forms using JSON editor, and we plan to add more features in future phases.

The APIs are onboarded and the tool is available from 
```SunbirdEd-Portal - 5.1.0 (@project-sunbird/form-service": "0.0.15")```

**Links:**
https://sb-forms.vercel.app/form-list
https://sb-forms.vercel.app/help

If you have any questions or need assistance getting started, please reach out to @rajeevsathish @sharathkashyap @rajeshkumaravel

https://user-images.githubusercontent.com/16378062/207589786-97d464d4-d860-4257-8e7c-96ec78ea7dec.mp4

---
@vinukumar-vs @Krishnaj20 @parthlawate @coolbung @reshmi-nair @pallakartheekreddy @swayangjit

**Comment by Krishnaj20** [comment]
tagging @rhwarrier @SanthoshVasabhaktula

**Reply by SanthoshVasabhaktula** [reply]
@rajeshkumaravel - Can you please upload a demo video for the tool?

**Reply by rajeshkumaravel** [reply]
@SanthoshVasabhaktula Sure

**Reply by nidhikachhawa24** [reply]
@rajeevsathish @sharathkashyap @rajeshkumaravel
while trying to login using domain url and bearer token, it say's **something went wrong**

**I'm using the following**:
Domain url - https://dhruva.shikshalokam.org/
Token - {bearer token}

It would be helpful if you can guide

**Reply by parthlawate** [reply]
Does it expect a certain version to be used ?

**Reply by rajeshkumaravel** [reply]
> @rajeevsathish @sharathkashyap @rajeshkumaravel while trying to login using domain url and bearer token, it say's **something went wrong**
> 
> **I'm using the following**: Domain url - https://dhruva.shikshalokam.org/ Token - {bearer token}
> 
> It would be helpful if you can guide

@nidhikachhawa24 Token value should be prefix with Bearer
ex: `Bearer eyJhbGciOiJI**********`

**Reply by rajeshkumaravel** [reply]
> Does it expect a certain version to be used ?

@parthlawate Yes; Expected SunbirdEd Portal version to be latest 5.1.0

**Reply by Krishnaj20** [reply]
tagging @anandvarada to this thread.

**Reply by Krishnaj20** [reply]
@rajeshkumaravel - the video looks great! thanks for sharing it with the community.
